### ISSUE TITLE
truncate longer attribute names

### ISSUE THREAD

**Issue by kjaymiller** [Python-Community-News/GH-Issues#14]
I'm not sure if this is something that I should even consider. I know that text is cheap, but long headers will result in long names...

My first consideration was to stop at the 3rd word break:

If the issue header is

```md
## This is a really long header
some text
```
the object attribute would be 
```python
Issue().this_is_a
```
an option would be to add `first_n` and `last_n` that would provide logic for truncating to the first/last `n` segments.

**Comment by blulady** [comment]
Yeah, this is a tough one... Would removing the vowels and capitalizing the first letter instead of the underscore make the linter angry? Otherwise your idea original idea is probably good.

**Comment by kjaymiller** [comment]
I don't like the idea of removing letters... Naming is hard and I think if you are working with a template enough you will have an idea of if you would like to truncate by the first or last whatever. Of course this is also a justification to just leave it as is.
